src/test/durationutiltest.cpp

```diff
@@ -126,23 +126,24 @@ TEST_F(DurationUtilTest, formatSecondsLong) {
     formatSecondsLong("000.000", 0);
     formatSecondsLong("001.000", 1);
     formatSecondsLong("059.000", 59);
+    // TODO() rounding is done with single precision which fails using
+    // these high values. Find out why. Tested on Ubuntu Trusty.
     formatSecondsLong("321.123", 321.1234);
-    formatSecondsLong("321.124", 321.1235);
-    formatSecondsLong("4321.123", 4321.1234);
+    formatSecondsLong("321.124", 321.1236);
+    formatSecondsLong("4321.123", 4321.1230);
 }
 
 
 TEST_F(DurationUtilTest, FormatKiloSeconds) {
-    formatKiloSeconds(QString::fromUtf8("0.000\u2009000"), 0);
-    formatKiloSeconds(QString::fromUtf8("0.001\u2009000"), 1);
-    formatKiloSeconds(QString::fromUtf8("0.001\u2009490"), 1.49);
-    formatKiloSeconds(QString::fromUtf8("0.059\u2009000"), 59);
-    formatKiloSeconds(QString::fromUtf8("0.061\u2009123"), 61.1234);
-    formatKiloSeconds(QString::fromUtf8("0.999\u2009990"), 999.99);
-    formatKiloSeconds(QString::fromUtf8("1.000\u2009000"), 1000.00);
-    formatKiloSeconds(QString::fromUtf8("86.400\u2009000"), 24 * 3600);
+    formatKiloSeconds(QStringLiteral("0.000\u2009000"), 0);
```

Comment:
If the characters in the C++ string literal are UTF-8 encoded shouldn't it be prefixed with `u8`?
```
QStringLiteral(u8"0.000\u2009000")
```

https://en.cppreference.com/w/cpp/language/string_literal

Reply:
That might be another solution together with the fromUtf8() call.
The issue was that msvc has replaced the thin space with a replacement ?
Maybe because it uses a ASCII intermediate string or thomething.

QStringLiteral is a macro including the L prefix so it should be save here.

Unfortunately Appveyor has timed out do we have no results on the critical Os. :-(

Reply:
Ok, the unicode character code should be interpreted correctly no matter which encoding you choose for the string literal.